azext_edge/edge/providers/support/shared.py

```diff
@@ -4,9 +4,21 @@
 # Licensed under the MIT License. See License file in the project root for license information.
 # ----------------------------------------------------------------------------------------------
 
+from azext_edge.edge.common import OpsServiceType
 from knack.log import get_logger
 from .base import process_nodes, process_events, process_storage_classes
 
+# Map for extension labels
+EXTENSION_LABELS = {
```

Comment:
support/shared is not the appropriate module for these attributes - shared is intended to capture cluster components that are not associated to a particular AIO service.

Reply:
I could see extending EdgeResourceApi to take an optional common label str. We already take a dependency on & reference the respective EdgeResourceApi in support bundle modules.